docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Full Riser
+sidebar_position: 3
+---
+
+<center>
+  <img
+    src="/img/robot_images/dingo_images/dingo_pacs_2.png"
+    width="600"
+  />
+</center>
+
+## Sales Kits
+
+| Description                | CPR Item |
+| :------------------------- | :------: |
+| Kit, Riser Full—Dingo, 100 |  027032  |
+| Kit, Riser Full—Dingo, 200 |  027033  |
+| Kit, Riser Full—Dingo, 300 |  027034  |
+| Kit, Riser Full—Dingo, 400 |  027035  |
+| Kit, Riser Full—Dingo, 500 |  027036  |
+
+## Parts List
+
+| ID  | Description                                         |                         CPR item                         | Quantity |
+| :-: | :-------------------------------------------------- | :------------------------------------------------------: | :------: |
+|  1  | Plate—Integration, 510 X 430                        |   [026909](/assets/pdf/clearpath_robotics_026909.pdf)    |    1     |
```

Comment:
Is it the same riser plate for Dingo-D and Dingo-O? I was initially assuming that they were different (and that the one for Dingo-O would be larger), but maybe that is too many parts to carry in inventory.

Reply:
We have a common item for Dingo-D and Dingo-O. The plate is 6 PACS positions long, to match the Dingo-Differential's length. The Dingo-O is 7 PACS positions long.

You can see this difference on https://build.clearpathrobotics.com/